.github/workflows/build.yml

```
@@ -45,6 +52,13 @@ jobs:
             echo "build_leeway_target=$(echo "$PR_DESC" | sed -n -e 's/^.*leeway-target=//p' | sed 's/\r$//')"
             echo "with_large_vm=${{ contains(github.event.pull_request.body, '[X] /werft with-large-vm') }}"
             echo "publish_to_npm=${{ contains(github.event.pull_request.body, '[X] /werft publish-to-npm') }}"
+            echo "publish_to_jbmp=${{ contains(github.event.pull_request.body, '[X] /werft publish-to-jb-marketplace') }}"
+            echo "with_ws_manager_mk2=${{ contains(github.event.pull_request.body, '[X] with-ws-manager_mk2') }}"
+            echo "with_dedicated_emu=${{ contains(github.event.pull_request.body, '[X] with-dedicated-emu') }}"
```

Nit: Spell out the full name of the option

```suggestion
            echo "with_dedicated_emu=${{ contains(github.event.pull_request.body, '[X] with-dedicated-emulation') }}"
```